openedx/adg/lms/student/views.py

```diff
@@ -23,11 +26,15 @@ def get_context_data(self, **kwargs):
         context = super(AccountView, self).get_context_data(**kwargs)
 
         providers_context_data = self.get_social_oauth_providers_data()
+        profile_image = get_profile_image_urls_for_user(self.request.user).get('full')
+
         context.update({
             'password_change_request': reverse('password_change_request'),
             'account_delete_url': reverse('deactivate_logout'),
+            'profile_image': profile_image,
             **providers_context_data
         })
+        context.update(account_settings_context(self.request))
```

Comment:
Instead of adding another update call, we can simply do the following:
```
context.update({
            'password_change_request': reverse('password_change_request'),
            'account_delete_url': reverse('deactivate_logout'),
            'profile_image': profile_image,
            **providers_context_data,
            **account_settings_context(self.request)
})
```